conf/messages

```diff
@@ -68,9 +68,6 @@ error.divorce-reason.required=Select an answer for keeping your Marriage Allowan
 pages.form.field-required-alt.endReason=Select an answer for keeping your Marriage Allowance until the end of the tax year
 
 pages.form.extra-year.field-required=Tell us whether you would like to apply for the previous {0} to {1} tax year
-#TODO remove after 6 April 2020
-pages.form.field-required.year-2015-=Confirm whether you would like to apply for the previous 2015 to 2016 tax year
-pages.form.field-required.year-2016-=Confirm whether you would like to apply for the previous 2016 to 2017 tax year
```

Comment:
Don't think this line should be removed as this is cy-4

Reply:
I've put the messages back in.

Reply:
Ah I only meant the second one, the first one should be removed still

Reply:
Ah i see now. Removed the 2015 line.